test/trivial_auction.js

```diff
@@ -101,4 +101,28 @@ contract('TrivialToken - Auction tests', (accounts) => {
         await bidAuction();
         await finishAuction();
     })
+
+    it('check Auction buyout - one user 100% tokens', async () => {
```

Comment:
Here should be some information about expected result. Not just "Check X", but more like "Doing X causes Y". The same remark may be applied to other tests. For instance, here I would call it "should be in a AuctionFinished state after auction is started by 100% tokens holder"